src/main/java/race/InputView.java

```diff
@@ -0,0 +1,42 @@
+package race;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private final Scanner SCANNER = new Scanner(System.in);
+    private final String CAR_NAME_SPLIT_REGEX = "\\s*,\\s*";
```

Comment:
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
    private static final String CAR_NAME_SPLIT_REGEX = "\\s*,\\s*";
```
InputView에 상태가 필요해 보이지 않는 것 같아요